documentation/operators/src/testing/performance.md

```diff
@@ -0,0 +1,32 @@
+# Performance Testing
+
+> To configure your node for a testing event, visit [node setup page](node-setup.md).
+
+Nym Mixnet has been running on mainnet for quite some time. There is still work to be done in order for the network to meet its full potential - mass adoption of privacy through fully distributed Mixnet.
+
+Nym therefore calls out the decentralised community of operators to join a series of Performance testing events in order to **overall increase quality of Nym Mixnet**. The main takeaways of such event are:
```

Comment:
'Nym asks its decentralised community of operators to join a series of performance testing events in order to **increase the overall quality of the Mixnet**.'